sale_properties_dynamic_fields/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+==============================
+Sale properties dynamic fields
+==============================
+
+This module allows to dynamically draw properties groups on sale order lines.
```

Comment:
@gurneyalex 
For consistency this change would also imply the rename of ```draw_dynamically``` that I don't want to do at this moment.
Maybe this can be postpone it until 9.0 version with the porting to the new API, what do you think?